testing/gnome-menus/APKBUILD

```diff
@@ -0,0 +1,21 @@
+pkgname=gnome-menus
+pkgver=3.32.0
+pkgrel=0
+pkgdesc='Library for the Desktop Menu Specification'
+arch="all"
+url='https://www.gnome.org/'
```

Comment:
I think linking to the actual homepage of gnome-menus instead of just gnome would be better